bulbs/videos/templates/videos/popular-series.html

```diff
@@ -0,0 +1,26 @@
+{% load betty %}
+
+<div class="popular-series">
+  <header>
+    <h2>Popular Series</h2>
+    <div class="popular-series-pagination-button">
+      <previous-button
```

Comment:
Is this still going to be hidden on launch?  We should use `bulbs-carousel` for this as well